hangman.rb

```diff
@@ -0,0 +1,255 @@
+require 'artii'
+require 'colorize'
+
+class Hangman
+  attr_accessor :guesses, :word, :hidden_word, :wrongs
+
+  def initialize
+    @guesses = []
+    @word = "" #pick word random from dictionary
+    @hidden_word = []
+    @wrongs = 0
+    @fails = 0
+  end
+
+  def make_hidden_word
+    @word.length.times do @hidden_word << "____  "
+    end
+
+  end
+
+  def choose_word(dictionary)
+    @word = dictionary.sample.upcase
+  end
+
+  def replace_hidden_with_letter(letter)
+    word_index =(0 ... @word.length).find_all {|i| @word[i,1] == letter}
+    num_letters = word_index.length
+
+    num_letters.times do |this|
+      @hidden_word[word_index[this]] = letter
+    end
+  end
+
+
+  def letter_included(guess)
+    if @word.include?(guess)
+      puts "Good Guess"
+      # add letter to hidden_word
+      replace_hidden_with_letter(guess)
+      #################
+      @hidden_word
+    else # ascii minus something
+      @fails += 1
+    end
+  end # End letter_included
+
+
+  def pick_a_letter
+
+    while @word.split("") != @hidden_word
+
+      puts "pick a letter"
+      letter_guess = gets.chomp.capitalize
+
+      case
+      when !!(letter_guess =~ /[A-Z]/) && !@guesses.include?(letter_guess)
+        @guesses << letter_guess
+        letter_included(letter_guess)
+
+        case
+        when @word.split("") == @hidden_word
+          win = Artii::Base.new
+          puts win.asciify('Winner Winner').colorize(:magenta)
+          puts win.asciify(@word).colorize(:cyan)
+        when @fails == 1
+          puts "          one wrong
+        o8%8888,
```

Comment:
I like your ascii art, but having all this text in the middle of your program makes it hard to read and to work with. Would it be possible to move this data to a separate file, perhaps something like `art.rb`?